packages/story-editor/src/app/font/types.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import type { FontData, FontWeight } from '@googleforcreators/elements';
+
+export interface FontWeightOption {
+  name: string;
+  value: FontWeight;
+}
+
+export interface FontProviderState {
+  state: {
+    fonts: FontData[];
+    curatedFonts: FontData[];
+    customFonts: FontData[] | null;
```

Comment:
In which case is it `null`? Saw that it's set to `[]` in the initial `state`, should it be `null` there instead perhaps?

Reply:
Good point, fixed in 9cffebe989,

Reply:
It actually had to be `null`, otherwise we can't tell "custom fonts not loaded" from "no custom fonts". Now they're respectively `null` and empty list.